frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,44 @@
+.errorMessage {
+  padding: 0;
+  margin: 0;
+  margin-bottom: var(--font-size-xxs);
+  font-size: var(--font-size-xxs);
+  line-height: var(--font-size-xs);
+  color: var(--color-red);
+}
+
+.input {
```

Comment:
Also, this is why I suggested to first look into adding `reset` or `normalize` since a lot of these styles would be handled already.

Reply:
Beatriz just added them in a separated PR, we can rebase as soon as she merges it @RicardoVaz1